File: app/src/main/java/com/ipd/taxiu/ui/activity/balance/BalanceBillActivity.java
package com.ipd.taxiu.ui.activity.balance;

import android.os.Bundle;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;

import com.ipd.taxiu.R;
import com.ipd.taxiu.adapter.BalanceBillAdapter;
import com.ipd.taxiu.ui.BaseUIActivity;
import com.ipd.taxiu.ui.fragment.balance.BalanceBillFragment;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by Miss on 2018/7/30
 * 余额账单
 */
public class BalanceBillActivity extends BaseUIActivity{
    @Override
    protected int getContentLayout() {
        return R.layout.activity_container;
    }

    @Override
    protected void initView(@Nullable Bundle bundle) {
        initToolbar();
    }

    @Override
    protected void loadData() {
     getSupportFragmentManager().beginTransaction().replace(R.id.fl_container, BalanceBillFragment.newInstance()).commit();
    }

    @Override
    protected void initListener() {

    }

    @NotNull
    @Override
    protected String getToolbarTitle() {
        return "余额账单";
    }
}

File: app/src/main/java/com/ipd/rainbow/ui/activity/balance/BalanceBillActivity.kt
package com.ipd.rainbow.ui.activity.balance

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import com.ipd.rainbow.R
import com.ipd.rainbow.ui.BaseUIActivity
import com.ipd.rainbow.ui.fragment.balance.BalanceBillFragment

/**
Created by Miss on 2018/8/10
余额账单
 */
class BalanceBillActivity : BaseUIActivity() {
    companion object {
        fun launch(activity: Activity) {
            val intent = Intent(activity, BalanceBillActivity::class.java)
            activity.startActivity(intent)
        }
    }

    override fun getContentLayout(): Int = R.layout.activity_container
    override fun getToolbarTitle(): String = "余额账单"
    override fun initView(bundle: Bundle?) {
        initToolbar()
    }

    override fun loadData() {
        supportFragmentManager.beginTransaction().replace(R.id.fl_container, BalanceBillFragment.newInstance()).commit()
    }

    override fun initListener() {
    }
}